nilearn/glm/first_level/first_level.py

```diff
@@ -414,6 +414,9 @@ class FirstLevelModel(BaseGLM):
 
     """
 
+    def __str__(self):
+        return "First Level Model"
```

Comment:
Could "First Level Model" be defined as class constant in `FirstLevelModel` and that constant be returned here?

Reply:
Or create a `level` property for the model?